etg/window.py

```diff
@@ -136,6 +136,16 @@ def run():
     m1.find('externalLeading').out = True
 
     c.find('GetHandle').type = 'wxUIntPtr*'
+    c.find('GetHandle').detailedDoc = [
+        """The returned value is the platform-defined handle for the native \
+        window containing the widget. On Windows this is an HWND, on X11 it \
+        is the X window ID, on Wayland it is a wl_surface, etc.\n\n"""
```

Comment:
The weasely "etc" is because I'm not sure what wxWidgets on Cocoa returns. My added code for GTK on Mac returns NSWindow, but I just spent a long time in the wxWidgets source and can't figure out what GetHandle will return. The good thing is at least for Cocoa this PR doesn't affect the return value so existing code should still work the same.